pageserver/src/page_service.rs

```diff
@@ -709,16 +708,23 @@ impl Connection {
 
             walreceiver::launch_wal_receiver(&self.conf, timelineid, &connstr);
 
+            self.write_message_noflush(&BeMessage::CommandComplete)?;
+            self.write_message(&BeMessage::ReadyForQuery)
+        } else if query_string.starts_with(b"pg_list") {
+            let branches = page_cache::get_repository().get_branches().unwrap(); // TODO handle error
```

Comment:
Is there a disadvantage to unwrapping and letting the thread panic vs propagating an error up? I think we could make `process_query` and `run` return an anyhow::Result instead of a std::io::Result and remove many of the unwraps from this code

Reply:
Yeah, +1 on avoiding unwraps here